API/c/Inc/pin_defs.h

```diff
@@ -49,6 +56,8 @@ typedef struct pin_name {
 #define LED3 (pin_name) {GPIOE, GPIO_PIN_6}
 #define LED4 (pin_name) {GPIOE, GPIO_PIN_5}
 
+#define ADCPINF6 (adc_pin_name) { ADC3, ADC_CHANNEL_4, GPIOF, GPIO_PIN_6 }
```

Comment:
Is this the only ADC pin we have exposed/can use?

Reply:
No I had gone back and made edits to allow us to declare gpio pins in the def so that we could use more than 1. That being said only certain pins can be connected to certain adcs and thats all "look in the data sheet" stuff. Also I dont think in this config more than one pin can be polled by an adc at a time effectively limiting us to 3 pins. For current uses thats not important but we should look into it for the future.